caffe-grid/src/main/python/examples/README-coco.md

```diff
@@ -93,25 +93,23 @@ Steps to run the COCO dataset for Image Captioning
     pushd ${CAFFE_ON_SPARK}/data/
     ln -s ~/Python2.7.10 Python2.7.10
     unzip ${CAFFE_ON_SPARK}/caffe-grid/target/caffeonsparkpythonapi.zip
-    cat /tmp/coco/parquet/vocab/part* > vocab.txt
     rm -rf /tmp/coco/parquet/df_caption_results_train2014
     spark-submit --master ${MASTER_URL} \
     		 --conf spark.cores.max=${TOTAL_CORES} \
     		 --conf spark.task.cpus=${CORES_PER_WORKER} \    
     		 --conf spark.driver.extraLibraryPath="${DYLD_LIBRARY_PATH}:Python2.7.10/lib" \
     		 --conf spark.executorEnv.LD_LIBRARY_PATH="${DYLD_LIBRARY_PATH}:Python2.7.10/lib" \
-    		 --conf spark.pythonargs="-model /tmp/coco/parquet/lrcn_coco.model -imagenet lstm_deploy.prototxt -lstmnet lrcn_word_to_preds.deploy.prototxt -vocab vocab.txt -input /tmp/coco/parquet/df_embedded_train2014 -output /tmp/coco/parquet/df_caption_results_train2014" examples/ImageCaption.py
+    		 --conf spark.pythonargs="-model /tmp/coco/parquet/lrcn_coco.model -imagenet lstm_deploy.prototxt -lstmnet lrcn_word_to_preds.deploy.prototxt -vocab /tmp/coco/parquet/vocab/part-00000 -input /tmp/coco/parquet/df_embedded_train2014 -output /tmp/coco/parquet/df_caption_results_train2014" examples/ImageCaption.py
     popd
-##### (7) Launch IPython Notebook
+##### (6 b) Launch IPython Notebook
     export IPYTHON_OPTS="notebook --no-browser --ip=127.0.0.1"
     pushd ${CAFFE_ON_SPARK}/data/
     ln -s ~/Python2.7.10 Python2.7.10
     unzip ${CAFFE_ON_SPARK}/caffe-grid/target/caffeonsparkpythonapi.zip
-    cat /tmp/coco/parquet/vocab/part* > vocab.txt
     pyspark --master ${MASTER_URL} --deploy-mode client \    
     	    --conf spark.driver.extraLibraryPath="${DYLD_LIBRARY_PATH}:Python2.7.10/lib" \
 	    --conf spark.executorEnv.LD_LIBRARY_PATH="${DYLD_LIBRARY_PATH}:Python2.7.10/lib" \
-	    --files "${CAFFE_ON_SPARK}/data/lstm_deploy.prototxt,${CAFFE_ON_SPARK}/data/vocab.txt,${CAFFE_ON_SPARK}/data/lrcn_word_to_preds.deploy.prototxt,${CAFFE_ON_SPARK}/data/caffe/_caffe.so,${CAFFE_ON_SPARK}/data/bvlc_reference_net.prototxt,${CAFFE_ON_SPARK}/data/bvlc_reference_solver.prototxt,${CAFFE_ON_SPARK}/data/lrcn_cos.prototxt,${CAFFE_ON_SPARK}/data/lrcn_solver.prototxt" \
+	    --files "${CAFFE_ON_SPARK}/data/lstm_deploy.prototxt,${CAFFE_ON_SPARK}/data/vocab.txt/part-00000,${CAFFE_ON_SPARK}/data/lrcn_word_to_preds.deploy.prototxt,${CAFFE_ON_SPARK}/data/caffe/_caffe.so,${CAFFE_ON_SPARK}/data/bvlc_reference_net.prototxt,${CAFFE_ON_SPARK}/data/bvlc_reference_solver.prototxt,${CAFFE_ON_SPARK}/data/lrcn_cos.prototxt,${CAFFE_ON_SPARK}/data/lrcn_solver.prototxt" \
```

Comment:
Don't you need to specify an alias for "${CAFFE_ON_SPARK}/data/vocab.txt/part-00000"? Otherwise, you will see a file named part-00000 in home dir.